app/uk/gov/hmrc/cataloguefrontend/metrics/model/Group.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.metrics.model
+
+//todo: should it be called DependencyGroup or similar?
+case class Group(name: GroupName, repos: Seq[RepositoryName], dependencies: Seq[DependencyName])
+
+object Group{
+
+  def apply(metrics: Seq[ServiceProgressMetrics]): Seq[Group] = metrics
```

Comment:
use of `apply` which doesn't return instances of `Group` (here `Seq[Group]`) usually leads to confusion - consider an alternative name.

Reply:
I think that's a bit subjective. I have seen apply methods returning anything from `Either` to `Validated` to `Option`. I don't think `Seq` is any different?